### PR TITLE
Add Method for GChat Notification

### PR DESCRIPTION
**Pull request by ckulal**
Add Method for GChat Notification

Signed-off-by: ckulal <ckulal@redhat.com>

# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs

# Checklist:

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarin Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test

### CODE REVIEW

File: pipeline/vars/common.groovy [hunk 1]
```diff
@@ -238,6 +238,18 @@ def fetchEmailBodyAndReceiver(def test_results, def isStage) {
     return ["to_list" : to_list, "jobStatus" : jobStatus, "body" : body]
 }
 
+def sendGChatNotification(def tier){
```
Comment (sunilkumarn417) [inline pipeline/vars/common.groovy:241]: Thanks for Adding this Notification service.

File: pipeline/vars/common.groovy [hunk 2]
```diff
@@ -238,6 +238,18 @@ def fetchEmailBodyAndReceiver(def test_results, def isStage) {
...
+    /*
+        Send a GChat notification.
+    */
+    currentBuild.result = currentBuild.currentResult
+    def msg= "Run for ${env.composeId}:${tier} is done"
```
Comment (sunilkumarn417) [inline pipeline/vars/common.groovy:246]: IMO, `STABLE/UNSTABLE/ABORTED` would be very clear on any run.

**Review by sunilkumarn417 — COMMENTED** [review]
Thanks for this PR. few minor comments.

Can you please add GoogleChatNotification used plugin description here.

**Comment by ckulal** [comment]
Raised New PR #724

**Pull request closed without merging**